server.js

```diff
@@ -1,12 +1,21 @@
 const axios = require('axios');
-const dotenv = require('dotenv').config();
+const dotenv = require('dotenv').config({path: __dirname + '/.env'});
 const express = require('express');
 const path = require('path');
 
 const app = express();
 const port = 1234;
 
-// console.log(process.env.GITHUB_TOKEN);
+const atelierHeaders = {
```

Comment:
This solution looks like it will work for handling our different authorization needs.